src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -140,7 +142,23 @@ export const ImportBlueprintModal: React.FunctionComponent<
     <Modal
       variant={ModalVariant.medium}
       isOpen={isOpen}
-      title={'Import pipeline'}
+      title={
+        <>
+          Import pipeline{' '}
+          <Popover
+            bodyContent={
+              <div>
+                You can import the blueprints you created by using the Red Hat
+                image builder into Insights images to create customized images.
+              </div>
+            }
+          >
+            <OutlinedQuestionCircleIcon
+              style={{ fontSize: 'var(--pf-v5-global--icon--FontSize--md)' }}
+            />
```

Comment:
We're using `<HelpIcon />` for popover buttons throughout the code base so even though the icons looks the same I'd vote for using the help one for the sake of consistency :sweat_smile: 

This is more of a suggestion so I'll leave the final decision up to you - we could use the PF Icon sizing to get rid of the inlined style like so:

```suggestion
            <Icon size="md">
              <HelpIcon />
            </Icon>
```